SDWebImage/UIView+WebCache.h

```diff
@@ -97,19 +97,23 @@ typedef void(^SDSetImageBlock)(UIImage * _Nullable image, NSData * _Nullable ima
 #pragma mark - Activity indicator
 
 /**
- *  Show activity UIActivityIndicatorView
+ Whether or not to show activity indicator
  */
-- (void)sd_setShowActivityIndicatorView:(BOOL)show;
+@property (nonatomic, assign) BOOL sd_showActivityIndicator;
 
 /**
- *  set desired UIActivityIndicatorViewStyle
- *
- *  @param style The style of the UIActivityIndicatorView
+ The desired activity indicator style
  */
-- (void)sd_setIndicatorStyle:(UIActivityIndicatorViewStyle)style;
+@property (nonatomic, assign) UIActivityIndicatorViewStyle sd_activityIndicatorStyle;
```

Comment:
These change to make the API more suitable for Swift and Objective-C user. Use method is little wired :)